src/components/Details/Details.tsx

```diff
@@ -18,11 +19,15 @@ const Details = ({ details }: { details?: GameDetails }) => {
     });
   }, []);
 
+  const toggleHeartStatus = (() => {
+    setHeartIconStatus(!heartIconStatus)
+  })
+
   return (
     <div>
       <h1></h1>
       <div className='large-font text-center'>
-        <BsSuitHeartFill className="heart" />
+        <BsSuitHeartFill className={heartIconStatus ? 'heart active' : 'heart'} onClick={toggleHeartStatus} />
       </div>
```

Comment:
Nice! We'll just have to swap out heartIconStatus with the correct property once it's merged.